homeassistant/components/fritz/common.py

```diff
@@ -241,3 +218,30 @@ def ip_address(self):
     def last_activity(self):
         """Return device last activity."""
         return self._last_activity
+
+
+class FritzBoxBaseEntity:
+    """Fritz host entity base class."""
+
+    def __init__(self):
+        """Init device info class."""
+
+    @property
+    def mac_address(self) -> str:
+        """Return the mac address of the main device."""
+        return self._fritzbox_tools.mac  # pylint: disable=maybe-no-member
+
+    @property
+    def device_info(self):
+        """Return the device information."""
+
+        return {
+            "connections": {(CONNECTION_NETWORK_MAC, self.mac_address)},
+            "identifiers": {
+                (DOMAIN, self.unique_id)
+            },  # pylint: disable=maybe-no-member
```

Comment:
We shouldn't need any of these pylint disables

Reply:
Without I get: `Instance of 'FritzBoxBaseEntity' has no 'unique_id' member (no-member)`
Safe to ignore ?

Reply:
Reverted